numba_dppy/dppy_lowerer.py

```diff
@@ -1165,6 +1167,9 @@ def lower(self):
         try:
             lowering.lower_extensions[parfor.Parfor].append(lower_parfor_rollback)
             self.gpu_lower.lower()
+            # if lower dont crash, and parfor_diagnostics is empthy then it is kernel
```

Comment:
```suggestion
            # if lower dont crash, and parfor_diagnostics is empty then it is kernel
```

Reply:
@reazulhoque you can propose code change via 
![image](https://user-images.githubusercontent.com/1541421/101601804-26cf3900-3a0e-11eb-841d-b85b94609299.png)